src/backend/settings/__init__.py

```diff
@@ -108,6 +108,13 @@
     "team.apps.TeamConfig",
     "sockets.apps.SocketsConfig",
     "stats.apps.StatsConfig",
+    "health_check",
+    "health_check.db",
+    "health_check.cache",
+    "health_check.storage",
+    "health_check.contrib.migrations",
+    "health_check.contrib.psutil",
+    "health_check.contrib.redis",
```

Comment:
We should probably include s3boto3_storage because we use that for file uploads iirc, maybe conditionally include it if including it by default causes issues with non-s3 setups?